commcare_connect/opportunity/models.py

```diff
@@ -406,6 +406,7 @@ class CompletedWork(models.Model):
     entity_id = models.CharField(max_length=255, null=True, blank=True)
     entity_name = models.CharField(max_length=255, null=True, blank=True)
     reason = models.CharField(max_length=300, null=True, blank=True)
+    status_modification_date = models.DateTimeField(null=True)
```

Comment:
`nit` I know the name came from @calellowitz but we tend to use verb-based such as `status_modified_date`

Reply:
Thanks @sravfeyn changed